.github/workflows/nodejs.yml

```diff
@@ -107,19 +138,19 @@ jobs:
           git config --global core.symlinks true
         if: runner.os == 'Windows'
       - uses: actions/checkout@v2
+      - name: Use Node.js 14.x
+        uses: actions/setup-node@v2.1.4
+        with:
+          node-version: 14.x
       - name: Get yarn cache
         id: yarn-cache
         run: echo "::set-output name=dir::$(yarn config get cacheFolder)"
       - uses: actions/cache@v2
         with:
           path: ${{ steps.yarn-cache.outputs.dir }}
-          key: ${{ runner.os }}-node-14.x-yarn-${{ hashFiles('**/yarn.lock') }}
+          key: yarn2-${{ hashFiles('yarn.lock') }}
           restore-keys: |
-            ${{ runner.os }}-node-14.x-yarn-
-      - name: Use Node.js 14.x
-        uses: actions/setup-node@v2.1.4
-        with:
-          node-version: 14.x
```

Comment:
Moved this as node should probably be configured before running yarn to get the cache path